hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmOzoneAclMap.java

```diff
@@ -298,4 +303,27 @@ public static OmOzoneAclMap ozoneAclGetFromProtobuf(
   public List<OzoneAclInfo> getDefaultAclList() {
     return defaultAclList;
   }
+
+  @Override
+  public Object clone() {
+    ArrayList<Map<String, BitSet>> accessMap = new ArrayList<>();
+
+    // Initialize.
+    for (OzoneAclType aclType : OzoneAclType.values()) {
+      accessMap.add(aclType.ordinal(), new HashMap<>());
+    }
+
+    // Add from original accessAclMap to accessMap.
+    for (OzoneAclType aclType : OzoneAclType.values()) {
+      final int ordinal = aclType.ordinal();
+      accessAclMap.get(ordinal).forEach((k, v) ->
+          accessMap.get(ordinal).put(k, (BitSet) v.clone()));
+    }
+
+    // We can do shallow copy here, as OzoneAclInfo is immutable structure.
+    ArrayList<OzoneAclInfo> defaultList = new ArrayList<>();
+    defaultList.addAll(defaultAclList);
+
+    return new OmOzoneAclMap(defaultList, accessMap);
+  }
```

Comment:
OmOzoneAclMap.clone() does not call super.clone(). Thus the correct object type is not guaranteed here and violates the contract for clone(). Most likely this is the findbug issue listed in CI run. To fix this, you must call the super.clone()

Reply:
I thought it is the final class and did like that. I think this should be marked as final like other classes like OMVolumeArgs. (As these classes represent proto structure data). And also as for few parameters, I need a deep copy, so I changed like this. Let me know what do you think.

Reply:
Given the use case and avoid shallow copy, it is better to make it final.